extensions/3DTILES_implicit_tiling/0.0.0/README.md

```diff
@@ -1,37 +1,739 @@
+<!-- omit in toc -->
 # 3DTILES_implicit_tiling
 
 **Version 0.0.0**, December 1, 2020
 
+<!-- omit in toc -->
 ## Contributors
 
 * Sam Suhag, Cesium
 * Sean Lilley, Cesium
 * Peter Gagliardi, Cesium
 * Erixen Cruz, Cesium
+* Patrick Cozzi, Cesium
```

Comment:
We should revisit the order of authors now that the text of this draft is written.